tests/integration/resources/test-pyspark-s3.py

```diff
@@ -0,0 +1,9 @@
+from random import random
+
+from pyspark.context import SparkContext
+from pyspark.sql.session import SparkSession
+
+sc = SparkContext()
+spark = SparkSession(sc)
+text_file = spark.read.text("s3a://test/example.txt")
+print(f"Number of lines {text_file.count()}")
```

Comment:
I believe this will only execute on the driver, not on the executors. It would be good to also test that all is working as it should on the executors too. You could do

```python
rdd = sc.parallelize([ "s3a://test/example.txt",  "s3a://test/example.txt" ])
rdd.map(path, spark.read.text(path).count()).collect()
```
Or something like that